docs/release-notes/mongoid-9.0.txt

```diff
@@ -40,3 +40,36 @@ any Mongoid-specific usages of this class, and change them to
 ``Mongoid::Errors::AttributeNotLoaded``. Note additionally that
 ``AttributeNotLoaded`` inherits from ``Mongoid::Errors::MongoidError``,
 while ``ActiveModel::MissingAttributeError`` does not.
+
+
+Use configured time zone to typecast Date to Time in queries
+-------------------------------------------------------------
+
+When querying for a Time field using a Date value, Mongoid now correctly
+considers the ``Mongoid.use_activesupport_time_zone`` to perform type
```

Comment:
```suggestion
considers the ``Mongoid.use_activesupport_time_zone`` configuration option to perform type
```
Or something like this... Make sure the line lengths add up correctly.

Reply:
Done